web-server/pages/api/internal/version.ts

```diff
@@ -75,24 +65,35 @@ interface DockerImage {
   last_pushed: string;
 }
 
-function readVersionFile(): ProjectVersionInfo {
-  const data = fs.readFileSync(versionFilePath, 'utf8');
-  const lines = data.split('\n').filter(Boolean);
-  const versionInfo: { [key: string]: string } = {};
-  lines.forEach((line) => {
-    const [key, value] = line.split(': ');
-    versionInfo[key] = value;
-  });
+interface TagCompressed {
+  name: string;
+  last_updated: string;
+  digest: string;
+}
+
+interface GitHubCommit {
```

Comment:
[optional]
intefaces not really needed, `type` will work just fine! unless you are using the special features interfaces provide